cmd/crowdsec-cli/utils.go

```diff
@@ -39,6 +40,36 @@ func indexOf(s string, slice []string) int {
 	return -1
 }
 
+func manageCliDecisionAlerts(ip *string, ipRange *string, scope *string, value *string) error {
+	log.Printf("checking stuff : ip:%s range:%s scope:%s value:%s", *ip, *ipRange, *scope, *value)
+	switch strings.ToLower(*scope) {
+	case "ip":
+		*scope = types.Ip
+		if *ip != "" {
+			*value = *ip
+		}
+		if *value != "" {
```

Comment:
may the `value`  variable be empty ? if yes what happen ?

Reply:
yes as it's used for `list` command as well, it's `add` and `delete` commands that check that it's coherent